packages/flutter/test/cupertino/button_test.dart

```diff
@@ -0,0 +1,101 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/scheduler.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Layout minimum size', (WidgetTester tester) async {
+    await tester.pumpWidget(
+      new Center(child: new CupertinoButton(child: new Text(' '), onPressed: null))
+    );
+    RenderBox buttonBox = tester.renderObject(find.byType(CupertinoButton));
+    expect(
+      buttonBox.size,
+      greaterThanOrEqualTo(const Size.square(48.0)),
```

Comment:
If you have two characters in the Ahem font at size 15, then the dimensions will be exactly 30x15, with the baseline at exactly 12 pixels from the top.

(This only works if the characters are in the Ahem font's character set, which covers most ASCII characters including in particular A-Z, a-z, 0-9, and space.)

In addition, the space character will be drawn as a transparent square, the É character will be drawn as fill above the baseline and transparent below, the p character will be drawn as transparent above the baseline and filled below, and the other characters will be drawn as filled squares.